kano_apps/AppGrid.py

```diff
@@ -51,8 +51,8 @@ def __init__(self, apps, main_win):
 
         last_page = 0
 
-        self._cat_names = [N_("latest"), N_("code"), N_("games"), N_("media"), N_("tools"),
-                           N_("others"), N_("experimental")]
+        self._cat_names = [_("latest"), _("code"), _("games"), _("media"), _("tools"),
+                           _("others"), _("experimental")]
```

Comment:
I think that translating these names at this stage will cause [this check](https://github.com/KanoComputing/kano-apps/blob/45bb1f7c320137107b138c01f810a41ad0b28545/kano_apps/AppGrid.py#L64-L66) to fail as the values of `app['categories']` will be in English but it will be comparing them against the translated versions.

Also, the [`used_categories`](https://github.com/KanoComputing/kano-apps/blob/45bb1f7c320137107b138c01f810a41ad0b28545/kano_apps/AppGrid.py#L61) is initialised with the English versions of `"latest"` and `"others"` so should the above check actually work then we would end up with `used_categories` containing a combination of English and translated strings which doesn't feel right.